paperqa/types.py

```diff
@@ -477,10 +493,13 @@ def populate_bibtex_key_citation(  # noqa: C901, PLR0912
                         data["other"]["bibtex_source"] = ["self_generated"]
                 else:
                     data["other"] = {"bibtex_source": ["self_generated"]}
-
-                existing_entry = next(
-                    iter(Parser().parse_string(data["bibtex"]).entries.values())
-                )
+                try:
+                    existing_entry = next(
+                        iter(Parser().parse_string(data["bibtex"]).entries.values())
+                    )
+                except PybtexSyntaxError:
+                    logger.warning(f"Failed to parse bibtex for {data['bibtex']}")
```

Comment:
```suggestion
                    logger.warning(f"Failed to parse bibtex for {data['bibtex']}.")
```

Periods help suss out empty string errors